shell/platform/tizen/tizen_renderer_evas_gl.cc

```diff
@@ -15,7 +15,9 @@ EVAS_GL_GLOBAL_GLES2_DEFINE();
 
```

Comment:
The comment at line 9 of this file needs an update.

Reply:
oh.. thank you for check. i updated comment